pkg/caretta/ebpf_map.go

```diff
@@ -0,0 +1,37 @@
+package caretta
+
+import "github.com/cilium/ebpf"
+
+type ConnectionsMapIterator interface {
+	Next(interface{}, interface{}) bool
+}
+
+type ConnectionsMap interface {
+	Lookup(interface{}, interface{}) error
+	Iterate() ConnectionsMapIterator
+	Delete(interface{}) error
+}
+
+type EbpfMap struct {
+	innerMap ebpf.Map
+}
+
+type EbpfMapItertor struct {
+	innerIterator ebpf.MapIterator
```

Comment:
I'm not sure copying iterators is an issue, but maybe use *ebpf.MapIterator to keep the original one.

Reply:
done